### DISCUSSION TITLE
How "unofficial" are these bindings?

### DISCUSSION THREAD

**Discussion by dufferzafar** [datafusion-contrib/datafusion-c#64] [General]
As compared to say the Python bindings which live under the `apache` organization: https://github.com/apache/arrow-datafusion-python

How "unofficial" are these bindings? What does it mean in terms of feature/support etc?

Want to try out datafusion over an in-memory arrow dataset in C++ and considering using this.

**Comment by kou** [comment]
In general, you can try this for it.
This doesn't update based DataFusion as much as the Python bindings does for now. Because I update the based DataFusion only when I have enough time.
If you contribute, we can update more frequently.
